unittests/test_policies.py

```diff
@@ -524,6 +525,8 @@ def test_kbd_interrupt_in_wait_with_limited_concurrency(
             KeyboardInterruptCheck(), SleepCheck(10),
             SleepCheck(10), SleepCheck(10)
         ]))
+        # FIXME: Dump everything in case Github #1369 appears
+        print(util.repr(runner))
```

Comment:
Unindent this and try running this unit test with `-s` to check what this innocent line prints ;-)